x-pack/plugins/secops/public/components/timeline/body/renderers/auditd_ended_session_details.test.tsx

```diff
@@ -0,0 +1,394 @@
+            args="arg1 arg2 arg3"
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual(
+        'Sessionsession-1[username-primary]@host-1inworking-directory-1ended fromprocess-1arg1 arg2 arg3'
+      );
+    });
+
+    test('it returns just a session if only given an id', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            userName={undefined}
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Session');
+    });
+
+    test('it returns only hostName if only hostname and an id is given', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            hostName="some-host-name"
+            userName={undefined}
+            secondary={undefined}
+            session={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Session@some-host-name');
+    });
+
+    test('it returns only a user name if only a user name and id is given', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            userName="some-user-name"
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processExecutable={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessionsome-user-name');
+    });
+
+    test('it returns only a process name if only given a process name and id', () => {
+      const wrapper = mountWithIntl(
+        <TestProviders>
+          <AuditdEndedSessionLine
+            id="hello-i-am-an-id"
+            userName={undefined}
+            processExecutable="some-process-name"
+            secondary={undefined}
+            session={undefined}
+            hostName={undefined}
+            primary={undefined}
+            processTitle={undefined}
+            workingDirectory={undefined}
+            args={undefined}
+          />
+        </TestProviders>
+      );
+      expect(wrapper.text()).toEqual('Sessionended fromsome-process-name');
+    });
+
+    test('it returns only session if process title with id is given', () => {
```

Comment:
If this description is correct, should the the expect match the expect on line [271](https://github.com/elastic/kibana/pull/33464/files#diff-ed21b2516e2913fae2036988ffae95a6R271), which only contains `session`:

```
expect(wrapper.text()).toEqual('Session');
```
? (The expect below, `expect(wrapper.text()).toEqual('Sessionsome-user-name');` currently includes the user name, similar to the expect [on line 311](https://github.com/elastic/kibana/pull/33464/files#diff-ed21b2516e2913fae2036988ffae95a6R311))

Reply:
It should return the user name. I will update the text.